monitor-template.md

```diff
@@ -0,0 +1,188 @@
+#test in github
+
+INFO TAB
+
+\<monitor logo>
+# \<Monitor name>
+
+Metadata associated with the \<name> collectd plugin can be found _here_. The relevant code for the plugin can be found _here_.
+
+- [Description](#description)
+- [Features](#features)
+- [Use](#use)
+- [Metrics](#metrics)
+- [Dimensions](#dimensions)
+
+## Description
+
+Monitors \<what> using the information provided by \<what> which collects metrics from \<what> instances by hitting these endpoints: 
+* <link to item>
+* <link to item>
+
+## Features
+
+For more information on the source data, see <https://...>
+
+Monitor Source Code \<link>
+
+Accepts Endpoints: <yes/no>
+
+Multiple Instances Allowed: <yes/no>
+
+
+## Use
+
+What the monitor is used for -- which metrics and for what -- discussion
+
+
+## Metrics
+
+Besides the common default metrics that are described [here](https://docs.signalfx.com/en/latest/integrations/agent/monitor-config.html), the following table shows additional optional metrics available for this monitor.
+
+- Metrics that are marked as Included are sent by default by the Smart Agent as part of a host-based subscription, and you are not charged for them.
+
+- Metrics that are not marked as Included are custom metrics, such as system or service metrics that you configure the Smart Agent to send outside of the default set of metrics. Your SignalFx subscription allows you to send a certain number of custom metrics.
+
+You may need to add a flag to these metrics. Check the config file for flag requirements.
+ 
+
+| Name | Type | Default | Description |
+| ---  | ---  | ---    | ---         |
+| `name1` | counter | ✔ | Total connections count per broker |
+| `name2` | gauge | ✔ | Total number of consumers subscribed to destinations on the broker |
+| `name3` | gauge |  | Total number of messages that have been acknowledged from the broker. |
+
+
+### Optional metric configuration
```

Comment:
Optional ==> Custom   

Refer to slack discussion in Integration Docs for wording to be used here

Reply:
Done.